sdk/python/tests/test_util.py

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import shutil
+import zipfile
+import yaml
+import tempfile
+import kfp_tekton.compiler as compiler
+import filecmp
+
+def _get_yaml_from_zip(zip_file):
+    with zipfile.ZipFile(zip_file, 'r') as zip:
+        with open(zip.extract(zip.namelist()[0]), 'r') as yaml_file:
+            return list(yaml.safe_load_all(yaml_file))
+
+
+def test_basic_workflow_without_decorator(test_data_dir):
```

Comment:
Can we generalize this helper function to take any pipeline without decorator? You can pass any infomation you need as arguments.